src/corehost/cli/deps_format.cpp

```diff
@@ -228,7 +226,8 @@ bool deps_json_t::process_runtime_targets(const json_value& json, const pal::str
                 if (pal::strcasecmp(type.c_str(), deps_entry_t::s_known_asset_types[i]) == 0)
                 {
                     const auto& rid = file.second.at(_X("rid")).as_string();
-                    assets.libs[package.first].rid_assets[rid].by_type[i].vec.push_back(file.first);
+                    deps_asset_t asset(file.first, empty, empty);
```

Comment:
doesn't this need to be updated for the `runtimeTargets` support?

Reply:
Yes, thanks.